functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -295,8 +297,8 @@ function Install-DbaMaintenanceSolution {
             }
 
             if ((Test-Bound -ParameterName ReplaceExisting -Not)) {
-                $procs = Get-DbaModule -SqlInstance $server -Database $Database | Where-Object Name -in 'CommandExecute', 'DatabaseBackup', 'DatabaseIntegrityCheck', 'IndexOptimize'
-                $table = Get-DbaDbTable -SqlInstance $server -Database $Database -Table CommandLog -IncludeSystemDBs | Where-Object Database -eq $Database
+                $procs = Get-DbaModule -SqlInstance $server -Database $Database | Where-Object Name -In 'CommandExecute', 'DatabaseBackup', 'DatabaseIntegrityCheck', 'IndexOptimize'
+                $table = Get-DbaDbTable -SqlInstance $server -Database $Database -Table @('CommandLog', 'Queue', 'QueueDatabase') -IncludeSystemDBs | Where-Object Database -EQ $Database
```

Comment:
Unsure if your editor is doing it, but don't worry about modifying the `-in` to -In`. Our convention is to leave it lower case.

Reply:
It is from VS Code because the proper casing of those parameters is uppercase. They are documented as such and do PSSA rule is kicking in to correct it since we have it set to format on save for our workspace.

Reply:
👎 i changed it 😅